From 860a227277ab425b0377875adc7ff5bdf3fa206e Mon Sep 17 00:00:00 2001
From: "Maxin B. John" <maxin.john@intel.com>
Date: Fri, 13 Oct 2017 10:52:35 -0500
Subject: [PATCH] drmdevice.c: define _GNU_SOURCE

Include config.h to fix this build error with uclibc:

libdrm-2.4.66/tests/drmdevice.c: In function 'main':
libdrm-2.4.66/tests/drmdevice.c:96:60: error:
'O_CLOEXEC' undeclared (first use in this function)
fd = open(devices[i]->nodes[j],O_RDONLY | O_CLOEXEC, 0);

Upstream-Status: Pending

Signed-off-by: Maxin B. John <maxin.john@intel.com>

---
 tests/drmdevice.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/drmdevice.c b/tests/drmdevice.c
index 9dd5098..78fcc39 100644
--- a/tests/drmdevice.c
+++ b/tests/drmdevice.c
@@ -21,6 +21,7 @@
  *
  */
 
+#include <config.h>
 #include <errno.h>
 #include <stdio.h>
 #include <stdlib.h>
